### PR TITLE
Fix innoextract windows package.

### PR DESCRIPTION
**Pull request by ofry**
Related to:
https://github.com/dscharrer/innoextract/issues/76
https://github.com/dscharrer/innoextract/issues/99

https://github.com/dscharrer/innoextract/pull/114

### CODE REVIEW

**Comment by Biswa96** [comment]
Remove `SKIP` for the shasum of the patch file. Use `updpkgsums` to update the shasums.

**Comment by ofry** [comment]
@Biswa96 Done.

File: mingw-w64-innoextract/PKGBUILD [hunk 1]
```diff
@@ -18,11 +18,16 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
...
+source=("https://constexpr.org/innoextract/files/${_realname}-${pkgver}.tar.gz"
+        0001-fix-segfault-windows.patch
+)
+sha256sums=('5e78f6295119eeda08a54dcac75306a1a4a40d0cb812ff3cd405e9862c285269'
+            'BFBDF47059B1EFDD74F7847679B0ED9B61E00E8A6236BA518C050B43C4D09D2A'
+)
 
 prepare() {
   cd "${srcdir}/${_realname}-${pkgver}"
+  patch -p1 -l -i "${srcdir}"/0001-fix-segfault-windows.patch
```
Comment (lazka) [inline mingw-w64-innoextract/PKGBUILD:30]: Please add a link to https://github.com/dscharrer/innoextract/pull/114 as a comment

Comment (ofry) [reply mingw-w64-innoextract/PKGBUILD]: Done.

**Comment by ofry** [comment]
@lazka done.

**Pull request merged**